sigstore/dsse.py

```diff
@@ -256,6 +274,18 @@ def _sign(key: ec.EllipticCurvePrivateKey, stmt: Statement) -> Envelope:
     )
 
 
+def _sign_envelope(
+        key: ec.EllipticCurvePrivateKey, envelope: Envelope) -> Envelope:
+    """
+    Sign the given envelope's payload and set the signature field
+    with the generated signature.
+    """
+    pae = _pae(envelope._inner.payload_type, envelope._inner.payload)
+    signature = key.sign(pae, ec.ECDS(hashes.SHA256()))
```

Comment:
Typo?

```suggestion
    signature = key.sign(pae, ec.ECDSA(hashes.SHA256()))
```

Reply:
Sorry! I thought the test ran but apparently it didn't.